hapi-proto/src/main/java/com/hederahashgraph/fee/FeeBuilder.java

```diff
@@ -449,13 +424,15 @@ public static int getBaseTransactionRecordSize(TransactionBody txBody) {
     if (txBody.hasCryptoTransfer()) {
       txRecordSize = txRecordSize
           + txBody.getCryptoTransfer().getTransfers().getAccountAmountsCount()
-          * (BASIC_ACCT_AMT_SIZE);
+          * (BASIC_ACCOUNT_AMT_SIZE);
     }
     return txRecordSize;
   }
 
   public static long getTxRecordUsageRBH(TransactionRecord txRecord, int timeInSeconds) {
-	if(txRecord == null) return 0;
+	if(txRecord == null) {
```

Comment:
Need to check code alignment in all the modified files. In many files the code is not aligned.

Reply:
@Neeharika-Sompalli will created a separate issue to track this for another PR https://github.com/hashgraph/hedera-services/issues/582

Reply:
Sounds good. Thanks @anighanta !